templates/js/non-sso-tab/src/views/hello.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="/favicon.ico" />
     <link rel="stylesheet" type="text/css" href="/static/styles/custom.css" />
     <script
-      src="https://res.cdn.office.net/teams-js/2.17.0/js/MicrosoftTeams.min.js"
+      src="https://res.cdn.office.net/teams-js/2.19.0/js/MicrosoftTeams.min.js"
       integrity="sha384-xp55t/129OsN192JZYLP0rGhzjCF9aYtjY0LVtXvolkDrBe4Jchylp56NrUYJ4S2"
```

Comment:
I think the integrity should be updated, too.

Reply:
Sorry I forgot to update integrity